ModelicaTest/Utilities.mo

```diff
@@ -553,10 +560,13 @@ extends Modelica.Icons.ExamplesPackage;
   model TestFiles
     extends Modelica.Icons.Example;
 
+    parameter String dirPath = Modelica.Utilities.Files.loadResource("modelica://ModelicaTest/Resources/Data");
+    parameter String filePath = Modelica.Utilities.Files.loadResource("modelica://ModelicaTest/Resources/Data/ReadStreamTestFile.txt");
+
```

Comment:
Arguments are redundant here since identical to the default formal arguments of function Files.